deployment/scripts/sonarqube.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "SonarQube Scan"
+echo "Directory: $PWD"
+echo "JAVA_HOME: $JAVA_HOME"
+
+mkdir $PWD/sonarqube/
+mkdir $PWD/sonarqube/download/
+mkdir $PWD/sonarqube/extract/
+mkdir $PWD/sonarqube/certs/
+mkdir $PWD/sonarqube/store/
+
+curl -o $PWD/sonarqube/certs/RH-IT-Root-CA.crt --insecure $RH_IT_ROOT_CA_CERT_URL
```

Comment:
Is this `--insecure` necessary?

Reply:
I don't think so, Just tried both ways. I'll remove it.